internal/metrics/metrics.go

```diff
@@ -26,6 +40,7 @@ func NewZerologMetrics(target string) Writer {
 		if err != nil {
 			return nil
 		}
+		defer file.Close()
```

Comment:
This change creates some issues with timestamps. The timestamp file is not getting written. It seems the file is closed at the end of the function execution and therefore, we can not write to the file.

Reply:
Thanks for pointing it out. I have removed the defer statement.